.circleci/config.yml

```diff
@@ -198,7 +224,20 @@ jobs:
       - run:
           name: Testing integrations
           command: |
-                INTEGRATION=1 go test -v -race -coverprofile=coverage.txt -covermode=atomic `go list ./contrib/... | grep -v -e grpc.v12 -e google.golang.org/api`
+            PACKAGE_NAMES=$(go list ./contrib/... | grep -v -e grpc.v12 -e google.golang.org/api | circleci tests split --split-by=timings --timings-type=classname)
+            gotestsum --junitfile ${TEST_RESULTS}/gotestsum-report.xml -- $PACKAGE_NAMES -v -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
This doesn't cover our special `Testing outlier google.golang.org/api` and `Testing outlier gRPC v1.2` steps right now. Not sure if this can be done easily given the way CircleCI handles JUnit XML results, but I could try to look into it if you think it's critical.